src/app/pages/system/failover-settings/failover-settings.component.ts

```diff
@@ -60,6 +68,7 @@ export class FailoverSettingsComponent implements OnInit {
     this.ws.call('failover.update', [values])
       .pipe(
         switchMap(() => {
+          this.store$.dispatch(loadHaStatus());
```

Comment:
As a rule of thumb ngrx actions are cheap to create and thus you sure create new actions instead of reusing old ones.
In here, it should dispatch something like `haSettingsUpdated()`